impls/gsnforwarder/forwarder_test.go

```diff
@@ -2,61 +2,37 @@ package gsnforwarder
 
 import (
 	"context"
-	"crypto/ecdsa"
 	"math/big"
 	"testing"
 
 	"github.com/ethereum/go-ethereum/accounts/abi/bind"
 	ethcommon "github.com/ethereum/go-ethereum/common"
-	ethcrypto "github.com/ethereum/go-ethereum/crypto"
-	"github.com/ethereum/go-ethereum/ethclient"
 	"github.com/stretchr/testify/require"
 
 	"github.com/athanorlabs/go-relayer/common"
+	"github.com/athanorlabs/go-relayer/tests"
 )
 
-// NODE_OPTIONS="--max_old_space_size=8192" ganache --deterministic --accounts=50
-var ganachePrivateKeys = []string{
-	"4f3edf983ac636a65a842ce7c78d9aa706d3b113bce9c46f30d7d21715b23b1d",
-	"6cbed15c793ce57650b9877cf6fa156fbef513c4e6134f022a85b1ffdd59b2a1",
```

Comment:
These same keys were being used in the `mforwarder` package and causing tests to fail. I added the same system we are using in atomic swap so different packages get different keys.